sub_crawler/__main__.py

```diff
@@ -14,9 +15,10 @@
 
 request = urllib.request.Request(url, headers=hdr)
 
-
-def crawl():
+@lru_cache(maxsize=1)
+def crawl(weeknr):
+    print("New Week!, getting a new list")
     page = urllib.request.urlopen(request)
-    soup = BeautifulSoup(page, 'html.parser')
-    rawData = soup.find_all('h3', {'class': 'menu-cat-prod-title'})
-    return list(map(lambda x: x.text, rawData))
+    rawData = BeautifulSoup(page, 'html.parser').find_all(
```

Comment:
waarom een find_all instead of map?

Reply:
find_all hadden we eerst ook, ik heb alleen de `soup` variable weg gehaald.
Ik heb ook de list(map(... vervangen door een list comprehension van gemaakt